examples/seq2seq/finetune.sh

```diff
@@ -10,8 +11,4 @@ python finetune.py \
     --do_predict \
     --n_val 1000 \
     --val_check_interval 0.1 \
-    --encoder_layerdrop 0.1 \
-    --decoder_layerdrop 0.1 \
-    --dropout 0.1 \
-    --attention_dropout 0.1 \
```

Comment:
Personally I would make `dropout = 0.1`, and other types of dropout = 0 (but still list them here). Just a suggestion. @sshleifer

Reply:
That resulted will be produced by `BartConfig`:
```
        encoder_layerdrop=0.0,
        decoder_layerdrop=0.0,
        attention_dropout=0.0,
        dropout=0.1,
```